src/news/newmaintainers.md

```diff
@@ -1,15 +1,15 @@
 ---
 title: New maintainers and step down.
-description: Swurl has stepped down from maintainer and a new core team has been established.
+description: Swirl has stepped down from maintainer and a new core team has been established.
 date: 2022-02-06
 tags:
   - News
 ---
 
-Based off recent contoversy lead maintainer swurl has decided to step down from their role as maintainer. Lenny is now the head maintainer of PolyMC and has established a new core group of maintainers to keep the project healthy and active.
+Based off recent contoversy lead maintainer Swirl has decided to step down from their role as maintainer. LennyMcLennington is now the head maintainer of PolyMC and has established a new core group of maintainers to keep the project healthy and active.
```

Comment:
make it "Based off of recent controversies, and for the project's best interest, Swirl has stepped down from his role as maintainer."